src/fsc/fsc.targets

```diff
@@ -54,4 +54,15 @@
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
   </ItemGroup>
 
+  <Target Name="Add PGO" DependsOnTargets="Get MIBC data" BeforeTargets="Publish" Condition="'$(Configuration)' == 'Release'">
```

Comment:
We need to ship fsc and fsi to the dotnetsdk as il assemblies not native compiled assemblies.

The sdk native compiles F# in this project here: https://github.com/dotnet/sdk/blob/main/src/Layout/tool_fsharp/tool_fsc.csproj

Reply:
So imagine, we will need to work with the sdk team to enable the pgo stuff ... at least if it relies on native compilation.

Reply:
We called certainly add the mibc files to the package we insert into the sdk.

Reply:
I don't think sdk does anything, @baronfel doesn't it just repacks stuff from nugets?

Reply:
I was looking at the `tool_fsc` (and also [this](https://github.com/dotnet/sdk/blob/034362b0af690d938b7affe13cecc4eff29395b8/src/Layout/redist/targets/GenerateLayout.targets#L276-L284)) and that was my impression.